### PR TITLE
Document the get_material_history method

### PR DESCRIPTION
**Pull request by bfolie**
Write a proper docstring for the method and include it in the "Getting Started" docs.

### CODE REVIEW

File: docs/source/getting_started/basic_functionality.rst [hunk 1]
```diff
@@ -67,9 +67,19 @@ There are currently three ways to filter, and they only work on data model objec
...
+
 Filtering by tags or attribute bounds can be scoped to a dataset or to a project.
 Filtering by name must be scoped to a dataset.
 
+Get Material History
+^^^^^^^^^^^^^^^^^^^^
+
+Starting with a specific root :class:`MaterialRun <citrine.resources.material_run.MaterialRun>`,
+you can retrieve the complete material history--every process, ingredient, material, and measurement
+that that went into the root material and that you have read access to. The method is
```
Comment (kroenlein) [inline docs/source/getting_started/basic_functionality.rst]: every process, ingredient and material that went into the root material .., as well as the measurements that were performed on all of those materials

?

File: src/citrine/resources/material_run.py [hunk 2]
```diff
@@ -95,8 +95,30 @@ def get_type(cls) -> Type[MaterialRun]:
...
         return MaterialRun
 
-    def get_history(self, scope, id):
-        """Get the history associated with a material."""
+    def get_history(self, scope, id) -> Type[MaterialRun]:
+        """
+        Get the history associated with a material.
+
+        The history contains every single process, ingredient, measurement, and material that
+        went into creating this particular material run. The returned object is a material run
```
Comment (kroenlein) [inline src/citrine/resources/material_run.py]: See above

File: docs/source/getting_started/basic_functionality.rst (hunk 1 above)
Comment (bfolie) [reply docs/source/getting_started/basic_functionality.rst]: Good suggestion, thanks

File: src/citrine/resources/material_run.py (hunk 2 above)
Comment (bfolie) [reply src/citrine/resources/material_run.py]: Same

**Pull request merged**